org.eclipse.xtext.builder.standalone/src/org/eclipse/xtext/builder/standalone/compiler/CompilerConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2013, 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.xtext.builder.standalone.compiler;
+
+/**
+ * @author Dennis Huebner - Initial contribution and API
+ */
+public class CompilerConfiguration {
+	private String sourceLevel = "1.5";
+
+	private String targetLevel = "1.5";
+
+	private boolean verbose;
```

Comment:
Not sure if there's a policy for xtext, but we like to explicitly set the initialized value on non final primitive fields

Reply:
I don't see a value in doing this for booleans.